stryker.conf.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "./node_modules/@stryker-mutator/core/schema/stryker-schema.json",
+  "packageManager": "npm",
+  "reporters": ["html", "clear-text", "progress"],
+  "mutate": ["lib/index.js"],
+  "testRunner": "command",
+  "coverageAnalysis": "off",
+  "commandRunner": {"command": "npm run test-api"},
```

Comment:
some test frameworks have coverage aware runners https://www.npmjs.com/search?q=%40stryker-mutator%20runner
`tape` currently isn't supported, so we fallback to the `command` runner which runs all tests every time